src/main/java/com/amihaiemil/docker/RtImage.java

```diff
@@ -47,14 +49,21 @@
      */
     private final URI baseUri;
 
+    /**
+     * The enclosing {@link Images} for this image.
+     */
+    private final Images parent;
```

Comment:
@llorllale I don't think this is correct. "parent Images" does not refer to the ``Images`` instance which created this Image, but to those ``Images`` that are parents in terms of history, previous versions of this ``Image``, see the ``history`` operation. Makes sense?

Reply:
@amihaiemil I see what you mean by "parent images"; I was just going for a "fluent" interface. I'll make it `void` then.